python_modules/libraries/dagster-dbt/dagster_dbt/asset_specs.py

```diff
@@ -60,6 +60,9 @@ def build_dbt_asset_specs(
         asset_out.to_spec(
             key=check.inst(asset_out.key, AssetKey),
             deps=[AssetDep(asset=dep) for dep in internal_asset_deps.get(output_name, set())],
+        )._replace(
```

Comment:
My main comment here is that there should be a comment here explaining why this is necessary.

Reply:
added